Form-Controls/index.html

```diff
@@ -1,27 +1,78 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="utf-8" />
-    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
-    <title>My form exercise</title>
-    <meta name="description" content="" />
-    <meta name="viewport" content="width=device-width, initial-scale=1" />
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>T-Shirt Order Form</title>
   </head>
   <body>
     <header>
-      <h1>Product Pick</h1>
+      <h1>Order Your T-Shirt</h1>
+      <p>Fill out the form below to select your T-shirt preferences.</p>
     </header>
     <main>
       <form>
-        <!-- write your html here-->
-        <!-- 
-          try writing out the requirements first as comments
-        this will also help you fill in your PR message later-->
+        <!-- Customer Name -->
+        <div>
+          <label for="customer-name">Name:</label>
+          <input
+            type="text"
+            id="customer-name"
+            name="customer-name"
+            minlength="2"
+            placeholder="Enter your full name"
+            required
+            aria-required="true"
+          />
+          <br><br>
+        </div>
+
+        <!-- Customer Email -->
+        <div>
+          <label for="customer-email">Email:</label>
+          <input
+            type="email"
+            id="customer-email"
+            name="customer-email"
+            placeholder="Enter your email address"
+            required
+          />
+          <br><br>
+        </div>
+
+        <!-- T-Shirt Colour -->
+        <div>
+          <label for="T-shirtcolour">Choose a T-Shirt Colour:</label>
+          <select id="T-shirtcolour" name="T-shirtcolour">
```

Comment:
I would tend to use `id` values with all lowercase letters. Using a capital like this is valid (there's nothing "wrong" with it as such!), but people expect them to be lowercase, so making everything lowercase makes them easier to remember.